src/gluonts/torch/modules/loss.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import torch
+
+
+class DistributionLoss(torch.nn.Module):
```

Comment:
Why do we have this base class?

Reply:
To make it explicit, when an object of type `DistributionLoss` is asked, that it should handle a `Distribution` as first argument to `forward`. Not having it would make this requirement a bit more implicit... but it's also not a big deal